L/libblastrampoline/common.jl

```diff
@@ -3,6 +3,8 @@
 using BinaryBuilder, Pkg
 using BinaryBuilderBase: sanitize
 
+Pkg.status()
```

Comment:
We have a [checked in manifest](https://github.com/JuliaPackaging/Yggdrasil/blob/fc4a8a977b075cb807b6016e99b137572c89416c/.ci/Manifest.toml), there isn't much need for this.